.github/workflows/ci.yml

```diff
@@ -83,7 +83,7 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --locked --all-targets --workspace --all-features
+          args: --locked --all-targets --workspace --all-features -- --include-ignored
```

Comment:
this is unfortunate, right now there's no effect but may cause issues in the future if more ignored tests are added

Reply:
Not sure the best way to solve this then.

Reply:
what about just requiring an env var to be set to disable, the nix builder can set the env var.